hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestDirectoryDeletingServiceWithFSO.java

```diff
@@ -534,12 +534,12 @@ private void assertTableRowCount(Table<String, ?> table, int count)
   private boolean assertTableRowCount(int expectedCount,
                                       Table<String, ?> table) {
     AtomicLong count = new AtomicLong(0L);
-    assertDoesNotThrow(() -> {
-      count.set(cluster.getOzoneManager().getMetadataManager().countRowsInTable(table));
+    assertDoesNotThrow(() -> count.set(cluster.getOzoneManager().getMetadataManager().countRowsInTable(table)));
+    try {
       LOG.info("{} actual row count={}, expectedCount={}", table.getName(),
           count.get(), expectedCount);
-    });
-
+    } catch (IOException e) {
+    }
```

Comment:
Thanks a lot @wzhallright for updating the patch.

So `table.getName()` can throw `IOException`?  In that case we should keep the previous version.  I'm sorry for the trouble.

Reply:
Done.